example/airship-example.cabal

```diff
@@ -25,3 +26,25 @@ executable example
                       , unordered-containers
                       , wai == 3.0.*
                       , warp == 3.0.*
+
+executable            versions
+  main-is:              Versions.hs
```

Comment:
Would it make sense to have a single example with separate contexts, or is it more digestible to see them in isolation? /noidea